apps/web/src/pages/ProfileAndApplicationsPage/components/TrackApplications/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { IntlShape } from "react-intl";
+import { parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import {
+  formatClosingDate,
+  formatSubmittedAt,
+  isDraft,
+} from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/utils";
+import { PoolCandidate } from "~/api/generated";
+
+export type Application = Omit<PoolCandidate, "user">;
+
+export const differenceInDays = (date1: Date, date2: Date): number => {
+  // Calculate the time difference in milliseconds
+  const timeDiff = date2.getTime() - date1.getTime();
+
+  // Convert milliseconds to days
+  const daysDiff = Math.floor(timeDiff / (1000 * 60 * 60 * 24));
+
+  return daysDiff;
+};
+
+export const isClosingSoon = (date: Date): boolean => {
+  const daysDiff = differenceInDays(new Date(), date);
+
+  return daysDiff <= 3;
+};
+export type ApplicationDateInfo = {
+  message: string;
+  color: string;
+  date: string;
+};
+
+export const getApplicationDateInfo = (
+  application: Application,
+  intl: IntlShape,
+): ApplicationDateInfo => {
+  const ApplyBy = intl.formatMessage({
+    defaultMessage: "Apply By: ",
+    id: "r4bJ8b",
+    description: "Label for showing the closing date of a job posting.",
+  });
+  const AppliedOn = intl.formatMessage({
+    defaultMessage: "Applied On: ",
```

Comment:
Should be sentence case. Also, a [dividing colon](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/packages/i18n/src/messages/commonMessages.ts#L82-L86) is available instead of using as part of the translation.

```suggestion
    defaultMessage: "Applied on: ",
```